disdrodb/L0/readers/ARM/ARM_ld.py

```diff
@@ -8,59 +8,27 @@
 
 import os
 import time
-import click
 
 # Directory
-from disdrodb.io import check_directories
-from disdrodb.io import get_campaign_name
-from disdrodb.io import create_directory_structure
+from disdrodb.L0.io import check_directories
+from disdrodb.L0.io import get_campaign_name
+from disdrodb.L0.io import create_directory_structure
 
 # Logger
-from disdrodb.logger import create_L0_logger
-from disdrodb.logger import close_logger
+from disdrodb.utils.logger import create_L0_logger
+from disdrodb.utils.logger import close_logger
 
 # Metadata
-from disdrodb.metadata import read_metadata
-from disdrodb.check_standards import check_sensor_name
+from disdrodb.L0.metadata import read_metadata
+from disdrodb.L0.check_standards import check_sensor_name
 
 # L0 processing
-from disdrodb.L0_proc import get_file_list
-from disdrodb.io import get_L1_netcdf_fpath
-from disdrodb.L1_proc import write_L1_to_netcdf
-
-# -------------------------------------------------------------------------.
-# CLIck Command Line Interface decorator
-@click.command()  # options_metavar='<options>'
-@click.argument("raw_dir", type=click.Path(exists=True), metavar="<raw_dir>")
-@click.argument("processed_dir", metavar="<processed_dir>")
-@click.option(
-    "-f",
-    "--force",
-    type=bool,
-    show_default=True,
-    default=False,
-    help="Force overwriting",
-)
-@click.option(
-    "-v", "--verbose", type=bool, show_default=True, default=False, help="Verbose"
-)
-@click.option(
-    "-d",
-    "--debugging_mode",
-    type=bool,
-    show_default=True,
-    default=False,
-    help="Switch to debugging mode",
-)
-@click.option(
-    "-l",
-    "--lazy",
-    type=bool,
-    show_default=True,
-    default=True,
-    help="Use dask if lazy=True",
-)
-def main(
+from disdrodb.L0.L0A_processing import get_file_list
+from disdrodb.L0.io import get_L1_netcdf_fpath
+from disdrodb.L0.L0B_processing import write_L0B
+
+
+def reader(
```

Comment:
You need to ensure that al readers have the same args now ! 
This is not the case

Reply:
Good point @ghiggi ! That could be our first test to setup. something like that : 

``` python

import inspect
from disdrodb.L0.retrive_reader import get_all_readers


list_of_expected_arguments = ['raw_dir', 'processed_dir', 'l0a_processing', 'l0b_processing', 'keep_l0a', 'force', 'verbose', 'debugging_mode', 'lazy', 'single_netcdf']


for data_source, values in get_all_readers().items() :
    for campaign_name, reader_file_path in values.items():
        full_name = f"disdrodb.L0.readers.{data_source}.{campaign_name}.reader"
        module_name, unit_name = full_name.rsplit(".", 1)
        my_function = getattr(__import__(module_name, fromlist=[""]), unit_name)

        list_of_current_arguments = inspect.getfullargspec(my_function)[0]

        assert list_of_expected_arguments == list_of_current_arguments
```

And we could go further by checking types...


I've corrected `ARM_lpm` and `ARM_ld`